test/plugin/test_out_gcplm.rb

```diff
@@ -0,0 +1,285 @@
+require 'test/unit'
+require "fluent/test"
+require 'fluent/test/driver/filter'
+
+require "../../lib/fluent/plugin/filter_gcplm"
```

Comment:
filter_gcp should be fine. We already have lm in the repo.
Also we don't need relative path here.
`./lib/fluent/plugin/filter_gcplm`